subgraph.mainnet.yaml

```diff
@@ -273,4 +273,23 @@ dataSources:
           handler: handleTransferWithReferenceAndFee
           receipt: true
       file: ./src/erc20EscrowToPay.ts
-  
\ No newline at end of file
+  - kind: ethereum/contract
```

Comment:
@mliu I don't know if the automated subgraph deployment will work with a file that is missing a contract address.  Can you remove this file until we have successfully deployed on Ethereum mainnet?

Reply:
Nvm. We deployed on mainnet.